task_1.js

```diff
@@ -0,0 +1,113 @@
+var express = require('express');
+var app     = express();
+var http    = require('http')
+
+var PATH = '/I/want/title';
+
+app.get(PATH, function (req, res) {
+
+  addresses = parseURL(req.url);
+  if(addresses){
+    titles  = [];
+    list_of_titles = '';
+
+    get_titles(addresses, function(titles){
+      console.log(titles);
+      res.send(list_of_titles);
+    });
+
+  }
+  else{
+    res.send('<h1>No address was present in the parameters');
+  }
+});
+
+app.listen(3000, function () {
+  console.log('Example app listening on port 3000!');
+});
+
+
+app.use(function(req, res, next) {
+  res.status(404).send('Sorry cant find that!');
+});
+
+
+function parseURL(req_url){
+  var url = require('url');
+  var url_parts = url.parse(req_url, true);
+  var query = url_parts.query;
+  if (query.address){
+    if(typeof(query.address) === 'string'){
+      return [query.address];
+    }
+    else{
+      return query.address;
+    }
+  }
+  else{
+    return null;
+  }
+}
+
+function get_titles(addresses, onComplete) {
```

Comment:
All other functions are using `CamelCase` notation. Why this method is in underscore mode.